src/platform/packages/shared/kbn-ui-theme/src/theme.ts

```diff
@@ -67,3 +67,13 @@ export const euiThemeVars: Theme = new Proxy(
     },
   }
 );
+
+export function getEuiThemeVars(theme: { name: string; darkMode: boolean }) {
+  return theme.name === 'borealis'
+    ? theme.darkMode
+      ? borealisDark
+      : borealisLight
+    : theme.darkMode
+    ? v8Dark
+    : v8Light;
+}
```

Comment:
nit: it took me a little to get the flow here. What if we make it more readable, even if not so concise? 

```suggestion
 export function getEuiThemeVars(theme: { name: string; darkMode: boolean }) {
  if (theme.name === `borealis`) {
    return theme.darkMode ? borealisDark : borealisLight;
  }
  return theme.darkMode ? v8Dark : v8Light;
}
```

Reply:
thanks for the suggestion, I've applied it here [`a40aa7b` (#206746)](https://github.com/elastic/kibana/pull/206746/commits/a40aa7bf20c6f74b05a11b0e7c3918424e70cc1d)